python/ray/job_configs.py

```diff
@@ -0,0 +1,45 @@
+import ray
+
+
+class JobConfigs:
```

Comment:
```suggestion
class JobConfig:
```

Reply:
Config is plural, you don't need the extra s. That would imply configs for multiple jobs.

Reply:
Good point! Done.